HousingRepairsOnlineApi/UseCases/ISendAppointmentConfirmationEmailUseCase.cs

```diff
@@ -1,10 +1,11 @@
-﻿using System.Threading.Tasks;
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
 using HousingRepairsOnlineApi.Domain;
 
 namespace HousingRepairsOnlineApi.UseCases
 {
     public interface ISendAppointmentConfirmationEmailUseCase
     {
-        public void Execute(string email, string bookingRef, string appointmentTime);
+        public void Execute(string email, Dictionary<string, dynamic> personalisation, string templateId);
```

Comment:
Use interface for parameter type 
```suggestion
        public void Execute(string email, IDictionary<string, dynamic> personalisation, string templateId);
```